ts/features/zendesk/screens/ZendeskAskPermissions.tsx

```diff
@@ -236,7 +247,12 @@ const ZendeskAskPermissions = (props: Props) => {
     .filter(it => it.value !== notAvailable);
 
   const handleOnCancel = () => {
-    openWebUrl(assistanceWebFormLink);
+    handleItemOnPress(
```

Comment:
What about the error scenario? 
If the users have no email client set, should we show a message error?

https://user-images.githubusercontent.com/822471/157081720-e28ef126-5f75-41cb-97a2-8eeef0ffd067.mov